content/soarian/dstu2.md

```diff
@@ -0,0 +1,69 @@
+
+The below figure shows the interaction summary between Application, IdP, Authorization server and Soarian API. 
+
+![Soarian API and Application interaction](/images/soarian-api-application-interaction.jpg)
+
+Once the IdP, Application, and API are registered, API is enabled and the user-patient-proxy identity and authorization is established, the patient/proxy Application user can use Application capabilities to query the API for:
+ * One or more C-CDA SOEN (consolidated clinical documentation architecture summary of episode note) containing an authorized patient's health information 
+ * Each category of health data such as a medication list, allergy list, problem list, diagnostic test results, or other specified information.  
+
+The Application is responsible for determining which of and how the API capabilities are exposed to the user. For each capability the Application offers the user, the FHIR resource pages define those API capabilities and, within each, which capabilities the Application must/shall support to be compliant with the API.
+
+## Special information regarding date parameters
+
+* All date formats are ISO 8601 formatted: YYYYMMDD.
+
+* Date is assumed formatted properly and always evaluated left to right: YYYYMMDD.  Note specifically that if 20152101 is used, MM is 21 or 12+9 months and the API will resove 20152101 to 20160901.
+
+* While DSTU2 uses a letter syntax for date (e.g., ge, le, eq), the API utilizes symbols (e.g., >=, <=, =).  Syntax for optional date parameter entry includes:
+
+* &date=\[YYYYMMDD].  Exactly and inclusive of the specified date.  Note that a date passed without a comparator (e.g., <) is interpreted as an exact date per FHIR DSTU 2.
+
+* &date=\[< YYYYMMDD].  Less than the date, not inclusive of the date.
+
+* &date=\[<= YYYYMMDD].  Less than or equal to the date, inclusive of the date.
+
+* &date=\[> YYYYMMDD].  Greater than the date, not inclusive of the date.
+
+* &date=\[>= YYYYMMDD].  Greater than or equal to the date, inclusive of the date.
+
+For the following range descriptions, entered sequence of high and low dates is immaterial, the API evalutes the symbols regardless of entry sequence.
```

Comment:
Or perhaps
"When searching with multiple dates, the dates must compose a closed range. The order that the bounds are provided in does not matter. Some examples of date range searches are below."